### PR TITLE
Add unix_seconds Spark function

### PR DESCRIPTION
**Pull request by acvictor**
This PR adds `unix_seconds` Spark function which returns the number of seconds since 1970-01-01 00:00:00 UTC.

Spark function doc - https://spark.apache.org/docs/latest/api/sql/index.html#unix_seconds

### CODE REVIEW

**Comment by acvictor** [comment]
@PHILO-HE can you please review this? Thank you!

File: velox/docs/functions/spark/datetime.rst [hunk 1]
```diff
@@ -244,6 +244,11 @@ These functions support TIMESTAMP and DATE input types.
         SELECT unix_date('1970-01-02'); -- '1'
         SELECT unix_date('1969-12-31'); -- '-1'
 
+.. spark:function:: unix_seconds(timestamp) -> bigint
+    Returns the number of seconds since 1970-01-01 00:00:00 UTC.::
+
+        SELECT unix_micros('1970-01-01 00:00:01'); -- 1
```
Comment (PHILO-HE) [inline velox/docs/functions/spark/datetime.rst:250]: typo: unix_micros -> unix_seconds.

**Review by PHILO-HE — COMMENTED** [review]
One comment. Thanks!

File: velox/docs/functions/spark/datetime.rst (hunk 1 above)
Comment (acvictor) [reply velox/docs/functions/spark/datetime.rst]: Thanks for the catch!

**Review by PHILO-HE — COMMENTED** [review]
Looks good! cc @rui-mo

**Review by rui-mo — COMMENTED** [review]
Looks good. Thanks!

**Comment by acvictor** [comment]
@mbasmanova can you please review this? Thank you!

**Comment by bikramSingh91** [comment]
will review it next week

**Comment by acvictor** [comment]
> will review it next week

Thank you!

**Review by bikramSingh91 — APPROVED** [review]

**Comment by bikramSingh91** [comment]
@acvictor Thank you for adding this. Can you please rebase and resolve conflicts so I can kick off the merge process. Thanks!

**Comment by acvictor** [comment]
> @acvictor Thank you for adding this. Can you please rebase and resolve conflicts so I can kick off the merge process. Thanks!

Thank you @bikramSingh91! Branch is rebased with main.

**Comment by acvictor** [comment]
@bikramSingh91 I rebased again, there were some conflicts.

**Pull request closed without merging**